cider-velox/test/e-2-e/presto/queries/tpcds/01.sql

```diff
@@ -0,0 +1,24 @@
+WITH customer_total_return AS
```

Comment:
Please add licenses for those files.

Reply:
Also, we need to add copyright here like https://github.com/intel/BDTK/blob/main/cider-velox/test/e-2-e/presto/cider-ut/CiderAggTest.sql#LL1C18-L1C39